google/auth/_default.py

```diff
@@ -79,7 +79,12 @@ def _warn_about_problematic_credentials(credentials):
 
 
 def load_credentials_from_file(
-    filename, scopes=None, default_scopes=None, quota_project_id=None, request=None
+    filename,
+    scopes=None,
+    default_scopes=None,
+    quota_project_id=None,
+    request=None,
+    use_project_from_env=False,
```

Comment:
I'm a little confused -- why are we passing this new var down? Don't we just need to skip the call in default if explicit_project_id is set?

Reply:
This method returns both the project and the cred itself. We still need the cred. We only do not want to use the cred to get project id, if already set in env.